lib/matplotlib/sphinxext/tests/tinypages/range4.py

```diff
@@ -0,0 +1,10 @@
+from matplotlib import pyplot as plt
+
+plt.figure()
+plt.plot(range(4))
+plt.show()
+
+def range6():
+    plt.figure()
+    plt.plot(range(6))
```

Comment:
The point is that plot 16 executes a specific function in a file. To make sure that works there needs to be some other stuff in the file that is not executed.
However, I might have to protect line 3-5 in some way to make sure they don't get executed on import. Maybe I can just make a function like this
```
def nevercalled():
    raise NotImplementedError
```